torch/testing/_internal/common_optimizers.py

```diff
@@ -1441,6 +1450,8 @@ def _get_optim_inputs_including_global_cliquey_kwargs(
             # Expected error between compiled forloop and fused optimizers
             DecorateInfo(
                 toleranceOverride({torch.float64: tol(atol=4.5e-7, rtol=2.2e-6)}),
+                "TestOptimRenewed",
+                "test_fused_matches_forloop",
                 active_if=lambda kwargs: is_dynamo_enabled()
```

Comment:
```suggestion
                active_if=is_dynamo_enabled()
```
should work too

Reply:
I'd recommend having a boolean in common_utils.py for IS_TESTING_DYNAMO that gets checked once vs getting evaluated at every call. Then we should use active_if=IS_TESTING_DYNAMO everywhere